app/models/tasks/copy_transcript_task.rb

```diff
@@ -0,0 +1,47 @@
+class Tasks::CopyTranscriptTask < ::Task
+  def transcript
+    owner
+  end
+
+  def source_url
+    transcript.href
+  end
+
+  def porter_tasks
+    [
+      {
+        Type: "Inspect"
+      },
+      {
+        Type: "Copy",
+        Mode: "AWS/S3",
+        BucketName: ENV["FEEDER_STORAGE_BUCKET"],
+        ObjectKey: porter_escape(transcript.path),
+        ContentType: "REPLACE",
+        Parameters: {
+          CacheControl: "max-age=86400",
+          ContentDisposition: "attachment; filename=\"#{porter_escape(transcript.file_name)}\""
+        }
+      }
+    ]
+  end
+
+  def update_transcript
+    transcript.status = status
+
+    if complete?
+      info = porter_callback_inspect
+      transcript.size = porter_callback_size
```

Comment:
I think this field is named `file_size` on this table, not `size`?

You'll also want to grab the `mime_type` (see `copy_media_task.rb`).

Have you tried actually running one of these through Porter?  Might be a good way to just validate everything is working.